erp/forms.py

```diff
@@ -339,48 +337,43 @@ class Meta:
         )
 
     fieldsets = schema.get_form_fieldsets()
+    nullable_values = [None, "", []]
+
+    def _build_data_for_field(self, field):
+        form_field = self[field["id"]]
+        field_value = form_field.value()
+        if field_value in self.nullable_values:
+            return
+        label, values = self.get_label_and_values(form_field.name, field_value, field["choices"], field["unit"])
+        return {
+            "name": form_field.name,  # TODO i am not sure we need the name
+            "label": label,
+            "value": field_value,
+            "values": values,
+            "is_comment": form_field.field.widget.template_name.endswith("textarea.html"),
+        }
 
-    def get_accessibilite_data(self, flatten=False):
+    def get_accessibilite_data(self):
         data = {}
-        for section, section_info in self.fieldsets.items():
-            data[section] = {
+        for section_name, section_info in self.fieldsets.items():
+            section_data = {
```

Comment:
It looks like section data is re-creating what is section_info, I believe we could skip this part but did not get it to work properly for now

Reply:
(I was missing the deepcopy, which led to the page working once but not twice)